README.md

```diff
@@ -0,0 +1,12 @@
+# CS4100 Artificial Intelligence
+### Final Project for Matthew DiLoreto and Kristofer Brethower
+
+This repository contains an agent to play the game [_"The Binding of Issac: Afterbirth+"_](https://store.steampowered.com/agecheck/app/570660/).
+
+## Installation and Usage
+1. Install _"The Binding of Issac: Afterbirth+"_ through Steam.
+2. Clone this repository to `C:/Users/[your username]/My Games/Binding of Isaac Afterbirth+ Mods/`
+3. On Steam, set --luadebug in launch options by right clicking the game and going to Properties -> General -> Set Launch Options
+4. Run Game
+
+By default the agent will begin to play, you can disable this by pressing "R"
```

Comment:
i switched it so that by default it doesn't play. Which do you prefer?

Reply:
No preference to be honest. We can leave the read me as is and resolve it when we have a real decision.

Reply:
deal